README.md

```diff
@@ -120,6 +120,85 @@ Both Congressional staff and the public would benefit from rapid access to Congr
 
 A JSON-formatted list of all Committee YouTube Channels can be found in this repo here: [Committee-Youtube-Channels.json](https://github.com/LibraryOfCongress/Congressional-Hackathon-2025/blob/main/Committee-Youtube-Channels.json).
 
+## 🏆 CapitolVoices - Real-Time Congressional Hearing Transcription
```

Comment:
Please don't add content to the root README.md file. Please move this to the README in your project folder.